NEW_README.md

```diff
@@ -0,0 +1,110 @@
+
+- Owner: @just-mitch
+- Approvers:
+  - @joeandrews
+  - @charlielye
+  - @iAmMichaelConnor
+  - @aminsammara
+  - @0xrafi
+  - @rahul-kothari
+  - @LHerskind
+  - @LeilaWang
+  - @dbanks12
+  - @PhilWindle
+  - @nventuro
+  - @Rumata888
+  - @critesjosh
+- Target PRD Approval Date: 2025-03-14
+
+## Key words
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Introduction
+
+- "Complex work" by engineering MUST have a Design Document (DD).
+- Determining whether work is "complex" is the responsibility of engineering team leads and product managers. Good candidates include:
+  - Introduces new concepts, data structures, algorithms, services, or contracts
+  - Interface or protocol changes
+  - Affects multiple products or users (including internal and external)
+  - Major refactoring or restructuring of code
+- A Design Document MUST have an associated Project Requirements Document (PRD).
+- PRDs and DDs MUST be kept up to date via pull requests.
+- Changes to the requirements in a PRD require re-approval by the individuals or groups specified in the original PRD.
+- Changes to the design which affect its compliance with the PRD MUST be re-approved by the individuals or groups specified in the original PRD.
+
+## Project Requirements Document (PRD)
+
+- A PRD MUST be created as a pull request.
+- A PRD MUST be a single document in a new directory in the `docs` directory.
+  - For example, if the project is called `cool-user-flow`, the PRD should be in `docs/cool-user-flow/prd.md`.
+  - A PRD MUST be written in markdown.
+- A PRD MUST identify:
+  - The project name
+  - The person responsible for the project
+  - Any individuals or groups that must approve the PRD.
+    - This MUST include
+      - someone on product
+      - an engineering team lead
+      - a devrel engineer if the project touches external stakeholders
+    - This MAY include
+      - someone from the legal team
+      - someone from the finance team
+      - someone from the sales team
+  - The PRD's target approval date
+  - The delivery deadline for the project
+  - The main user stories the project is intended to support
+  - How/If demand for those user stories has been measured
+  - Requirements describing the desired functionality/qualities in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) terms
+  - Each requirement SHOULD identify:
+    - Why the requirement exists
+    - Where the requirement comes from (e.g. what user is asking for it, or other observation inspired the requirement)
+    - The KPIs and target values that candidate solutions will be measured against
+  - Each requirement MAY:
+    - Give guidance on how the requirement may change beyond the delivery deadline
+- The PRD SHOULD:
+  - Define any key terms used in the requirements
```

Comment:
```suggestion
  - Define any key terms used in the requirements
  - Define key customers (with examples of their requirements)
```
^ this forces requirements to have been cleared by the users (like in the token case...)